sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/xml/XmlSuite.scala

```diff
@@ -2561,10 +2575,10 @@ class XmlSuite extends QueryTest with SharedSparkSession {
          |            </array1>
          |            value10
          |            <array1>
-         |                <struct3>
+         |                <struct3><!--A comment within tags-->
          |                    <array2>3</array2>
          |                    value11
-         |                    <array2>4</array2>
+         |                    <array2>4</array2><!--A comment within tags-->
          |                </struct3>
          |                <string>string</string>
          |                value12
```

Comment:
GitHub doesn't allow to add review comment to any arbirary line. So commenting here.

Add a comment between `value16` and `</ROW>` with a space before and after the comment.

Reply:
Added 👍